server/graphql-schema.js

```diff
@@ -396,7 +396,7 @@ const graphqlSchema = gql`
         """
         The AT mode the test was written to expect.
         """
-        atMode: AtMode!
+        atMode: AtMode
```

Comment:
I feel like having a MODELESS AtMode makes more sense than making this field nullable. Null to me implies more that the mode is missing or not relevant, and it would match the way we're handling modes in the database.

Reply:
This was a missed usage opportunity. But returning `MODELESS` may not be logical for some tests. How the tests are in the v2 test format now is that they can have multiple states per test whereas v1 had a defined `atMode` attribute so this would have to be the following instead:
```diff
-atMode: AtMode!
+atModes: [AtMode]!
```

That was already being handled because of how collected.json passes along the assertions data but there is a reason for extending that here, for sure.

Reply:
After an internal discussion and discussing some ways to best handle this, it makes more sense to just remove `atMode` from the `type Test` in graphql-schema.js. The field isn't being referenced on any pages and it's only been referenced when copy the value during the "test update" processes. When mode for a test is needed, it is already correctly referenced by the aria-at-harness from `renderableContent`.

Like my comment in https://github.com/w3c/aria-at-app/pull/840#discussion_r1396413513. I'd rather make this change in a follow up PR.

Reply:
I'm definitely happy with this outcome, nice idea Howard!